lib/install.ps1

```diff
@@ -44,6 +46,10 @@ function install_app($app, $architecture, $global, $suggested) {
     if($global) { ensure_scoop_in_path $global } # can assume local scoop is in path
     env_add_path $manifest $dir $global
     env_set $manifest $dir $global
+
+    # persist data
+    persist_data $manifest
```

Comment:
I think it would be safer and clearer to explicitly pass `$original_dir`, `$persist_dir` variables here, instead of having `persist_data` inherit them from the parent scope.